### ISSUE TITLE
browser_auth setting does not work in ~/snowflake/config

### ISSUE THREAD

**Issue by chriskl** [Snowflake-Labs/terraform-provider-snowflake#1881]
**Provider Version**

0.66.1

**Terraform Version**

v1.4.4

**Describe the bug**

If you use a `~/.snowflake/config file` like this:

```
account='BLAH'
username='BLAH'
browser_auth=true
```

It does not pop a browser auth and instead it complains that the 'Password is empty'

However if you use a `.env` file like this:

```sh
export SNOWFLAKE_USER='BLAH'
export SNOWFLAKE_ACCOUNT='BLAH'
export SNOWFLAKE_USE_BROWSER_AUTH=true
```

It works perfectly.

**Expected behavior**

Using the `~/.snowflake/config` file should work the same as environment variables and correctly popup browser auth.

**Code samples and commands**

`terraform plan`

**Comment by sfc-gh-asawicki** [comment]
The reasons are the same as presented in https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2145#issuecomment-2194752467. We may be able to address this during the upcoming https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#providers-configuration-rework.

**Comment by sfc-gh-jmichalak** [comment]
Hi @chriskl  👋

We've released a new v0.98.0 version ([release](https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.98.0), [migration guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0970--v0980)) with reworked config, which now supports setting authenticator in a TOML file. Please migrate to this version and adjust your configuration files.

**Comment by sfc-gh-jmichalak** [comment]
Closing due to inactivity. Please create a new issue if the problem persists in the newest version.
